packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet-item.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import { AppLinkedWithWalletIcon, Text, Badge } from '@deriv/components';
+import { useActiveAccount, useWalletAccountsList } from '@deriv/hooks';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import './account-switcher-wallet-item.scss';
+
+type TAccountSwitcherWalletItemProps = {
+    account: ReturnType<typeof useWalletAccountsList>['data'][number];
+    closeAccountsDialog: () => void;
+    is_dark_mode_on: boolean;
+    switchAccount: (loginid?: string) => Promise<void>;
+};
+
+export const AccountSwitcherWalletItem = ({
```

Comment:
Would be better to wrap it by `observer` and get `switchAccount` and `is_dark_mode_on` directly from useStore() hook.

Reply:
updated.